Polyhedron/demo/Polyhedron/Viewer.h

```diff
@@ -174,6 +175,7 @@ public Q_SLOTS:
   QString message;
   bool _displayMessage;
   QTimer messageTimer;
+  QMatrix4x4 projectionMatrix;
```

Comment:
I think the `Viewer` class should use a d-pointer, like the other ones in our framework, to avoid that modifications for internal reasons, like this one, have an impact on the `Viewer.h`.